dashboard/src/components/CommitNavigationGraph/CommitNavigationGraph.tsx

```diff
@@ -210,12 +210,11 @@ const CommitNavigationGraph = ({
                     })
                     .parse(possibleIndexNumber);
 
-                  const name = commitData[parsedPossibleIndex].commitName;
                   isCurrentCommit =
                     treeId === commitData[parsedPossibleIndex].commitHash;
-                  displayText = (
-                    name ?? commitData[parsedPossibleIndex].commitHash
-                  ).slice(0, graphDisplaySize);
+                  displayText = commitData[
+                    parsedPossibleIndex
+                  ].commitHash.slice(0, graphDisplaySize);
```

Comment:
```suggestion
                  ]?.commitHash.slice(0, graphDisplaySize);
```

arrays are unsafe in javascript